server/py/services/api/tests/unit/runtimes/test_spark.py

```diff
@@ -1139,7 +1139,7 @@ def test_get_offline_features(
                 target=ParquetTarget(),
             )
 
-        self.project = "default"
+        self.project = "active"
```

Comment:
this is the result of mass replace(?), a project named "active" is not very clear (in tests too).